.github/workflows/nightly-build-image.yml

```diff
@@ -1,101 +1,20 @@
 name: Build New Image Version
-on:
-  # Manually call
-  workflow_dispatch:
```

Comment:
You won't be able to manually trigger if there's no workflow_dispatch

Reply:
Got it. Will add back to test manually